docs/how-to-guides/data-basics/2-building-a-list-of-pages.md

```diff
@@ -273,10 +273,11 @@ A few messages like  _Loading…_ or _No results_ would clear it up. Let’s imp
 ```js
 import { SearchControl, Spinner } from '@wordpress/components';
 function PagesList( { hasResolved, pages } ) {
-	if ( !hasResolved ) {
-		return <Spinner/>
-	}
 	if ( !pages?.length ) {
+		if ( !hasResolved ) {
+			return <Spinner/>
+		}
```

Comment:
I find this code change strange unless we're going to come back later in the tutorial and clean it up. We took what were separate concepts (`no results` and `no results yet`) and mashed them together by nesting the `if's.

Did we intend on doing this? What was the problem with the early-abort style it had already?